config.md

```diff
@@ -26,7 +26,7 @@ Changing it means creating a new derived image, instead of changing the existing
 ### Layer DiffID
 
 A layer DiffID is the digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., `sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9`.
-Layers must be packed and unpacked reproducibly to avoid changing the layer DiffID, for example by using tar-split to save the tar headers.
+Layers must be packed and unpacked reproducibly to avoid changing the layer DiffID, for example by using [tar-split][tar-split] to save the tar headers.
```

Comment:
nit: You can just use `[tar-split][]` to take advantage of the [implicit link name shortcut][1].  Removing the redundant link name makes this line slightly easier to read, but it's obviously not a big deal either way.

[1]: https://daringfireball.net/projects/markdown/syntax.php#link